tests/modules/nf-core/kallisto/quant/main.nf

```diff
@@ -18,10 +18,11 @@ workflow test_kallisto_quant {
         ]
     ]
 
-    KALLISTO_INDEX ( fasta )
+    KALLISTO_INDEX ( [[:], fasta] )
```

Comment:
Do we need this file? We have `modules/nf-core/kallisto/quant/tests/main.nf.test` that should test the module directly? Would prefer to keep any nf-test updates in this release as minimal as possible so that we can add them in the next release

Reply:
Ah pants! I thought this PR was to nf-core/rnaseq 🤦🏽 Ignore both of my comments. I was wondering why I was seeing the diff in the above comment because it should be a brand new file if added to nf-core/rnaseq.

Maybe @adamrtalbot can check out the nf-test aspect. I thought we moved away from needing to add tests in `./tests/modules/nf-core/kallisto/quant` in favour of adding a `tests/` folder in the main module directory? Might be missing something here.

Reply:
Oh, I see. We are updating the tests for nf-test and pytest-workflow in this PR. Makes sense.